src/outfitMakerUse1.java

```diff
@@ -0,0 +1,73 @@
+import java.util.ArrayList;
+
+/**
+ * A sample use of the outfitMaker component (non-interactive)
+ */
+
+public class outfitMakerUse1 {
```

Comment:
By convention, we use UpperCamelCase for classes, so this should probably be `OutfitMakerUse1`